lib/asyncbox.js

```diff
@@ -43,34 +41,11 @@ async function retryInterval (times, sleepMs, fn, ...args) {
 }
 
 async function parallel (promises) {
-  if (!(promises instanceof Array)) {
-    throw new Error("Must parallelize a list of promises");
-  }
-  if (promises.length < 1) {
-    // no need to do anything for an empty set
-    return [];
-  }
-  let done = Q.defer();
-  let results = [];
-  for (let p of promises) {
-    /*jshint -W083 */
-    if (!p.then || !p.catch) {
-      throw new Error("Tried to parallelize a non-promise");
-    }
-    p.then(function (res) {
-      results.push(res);
-      if (results.length === promises.length) {
-        done.resolve(results);
-      }
-    }, function (err) {
-      done.reject(err);
-    });
-  }
-  return done.promise;
+  return await B.all(promises);
```

Comment:
https://github.com/petkaantonov/bluebird/blob/master/API.md#all---promise
Looks like it. I don't think the Q one has.

btw parallel is a bad name, the promises have already started. Usually control flow methods take function returning promises as arguments.